lib/system/chcks.nim

```diff
@@ -8,7 +8,7 @@
 #
 
 # Implementation of some runtime checks.
-import system/indexerrors
+include system/indexerrors
```

Comment:
Why?

Reply:
The reasoning was for ```system``` to not import anything (```chcks.nim``` is itself included in system). This is mostly because of the order of initialization of modules - up until this got introduced ```system``` didn't import anything. The difference is that if it is included like this, the contents of ```indexerrors``` will also be exported from ```system```...

I see that recently ```system``` started importing and immediately after that exporting 2 modules - ```widestrs``` and ```io```, but I didn't change them to being included. I might revisit this whole situation.

Reply:
System.nim must be able to import modules, we will depend more heavily on this feature in the future, but they can all be markes as ``.noReload`` or whatever you called it. ;-)